Second round of style conventions edits for: "How to Deploy an AnyTrust chain using the Orbit SDK" (#1161)
OffchainLabs/arbitrum-docs: arbitrum-docs/launch-orbit-chain/how-tos/orbit-sdk-deploying-anytrust-chain.md

### About AnyTrust Orbit

> AnyTrust chains follow a different protocol than Rollup chains, offering different features and deployment processes. For an overview of Orbit chain types, please refer to the [Orbit SDK introduction](../orbit-sdk-introduction.md) page.

Reviewer: Anytrust chains still follow the Rollup protocol, but have a different model around data availability, which leverages the additional Anytrust protocol

> AnyTrust chains follow the Rollup protocol but offer different Data model availability, features, and deployment processes. For an overview of Orbit chain types, please refer to the [Orbit SDK introduction](../orbit-sdk-introduction.md) page.

Reviewer 2: Any particular reason `Data` is capitalized here?

Author: No particular reason, thank you for spotting this one.

### Deployment steps 

The deployment process of AnyTrust chains is very similar to that of [Rollup chains](orbit-sdk-deploying-rollup-chain.md#rollup-config-param), but with some differences that we'll discuss in this guide.

Here are the steps involved in the deployment process:

1. **[Setting up the chain parameters](#1-setting-up-the-chain-parameters)**
2. **[Deploying your AnyTrust chain](#2-deploying-your-anytrust-chain)**
3. **[Getting the AnyTrust Orbit chain information after deployment](#3-getting-the-anytrust-orbit-chain-information-after-deployment)**
4. **[Setting a valid key set on parent chain](#4-setting-valid-key-set-on-parent-chain)**

> The deployment of an AnyTrust Orbit chain involves defining and setting up the <a data-quicklook-from="data-availability-committee-dac">`Data Availability Committee (DAC)`</a> keyset. This key set includes keys from the appointed members of the DAC. They are required to ensure the chain's data availability and integrity. Once you have selected your committee members and gathered their keys, the Orbit SDK helps you configure these keys into a keyset. 

Reviewer 2: flagging inconsistency between "keyset" / "key set" - if needed, we can use the [terminology guide](https://www.notion.so/arbitrum/Pattern-guide-Docs-dafaf8d1ce34433d88c1ef3b4b99c19c?pvs=4#92b5697554434a7ea09e1a2781aeda67) to align on & document consistent terminology patterns throughout our corpus

This key set is then embedded into the chain, serving as a verification mechanism.

Let's go through each deployment step:

### 1. Setting up the chain parameters 

Similarly to the Rollup chain, you'll need to prepare the AnyTrust chain configuration, including the core contracts and operational parameters that govern the chain's functionality, focusing on parameters specific to AnyTrust chains.

> ```solidity {10}

Reviewer 2: Probably a silly q, but what's this `{10}` for?

Author: It highlights line #10.

### About AnyTrust Orbit

> AnyTrust chains follow the Rollup protocol but offer different data model availability, features, and deployment processes. For an overview of Orbit chain types, please refer to the [Orbit SDK introduction](../orbit-sdk-introduction.md) page.

Reviewer 2: ```suggestion
AnyTrust chains implement the Arbitrum AnyTrust protocol, an alternative to the Arbitrum Rollup protocol. AnyTrust chains reduce transaction fees by introducing a minor trust assumption in the form of a permissioned set of parties responsible for managing data availability. For an overview of Orbit chain types, please refer to the [Orbit SDK introduction](../orbit-sdk-introduction.md).
```

| `MaxCodeSize  `             | Sets the maximum size for contract bytecodes on the Orbit chain. e.g. Ethereum mainnet has a limit of 24,576 Bytes.                             |
| `MaxInitCodeSize`           | Similar to `MaxCodeSize`, defines the maximum size for your Orbit chain's **initialization** code. For example, the Ethereum mainnet limit is 49,152 Bytes. |

For an AnyTrust chain, you need to set the `DataAvailabilityCommittee` to **true**. This setting is crucial as it indicates the chain's reliance on a committee for data availability.

Here's an example of how to configure the `chainConfig` for an AnyTrust chain using the Orbit SDK:

```js
import { prepareChainConfig } from "@arbitrum/orbit-sdk";

const chainConfig = prepareChainConfig({
  chainId: Some_Chain_ID,
  arbitrum: {
    InitialChainOwner: deployer_address,
    DataAvailabilityCommittee: true,
  },
});
```

In this example, you need to set up the chain configuration with a unique `chainId`, the `InitialChainOwner` as the deployer's address, and, importantly, you must set the `DataAvailabilityCommittee` as `true`.

> The Orbit SDK has a helper to execute this part: the `prepareChainConfig` API, which takes config parameters as arguments and returns a `chainConfig` `JSON` string. Any parameters not provided will default to standard values, which are detailed in the [Orbit SDK Repository](https://github.com/OffchainLabs/arbitrum-orbit-sdk/blob/1f251f76a55bc1081f50938b0aa9f7965660ebf7/src/prepareChainConfig.ts#L3-L31).

Reviewer 2: Any particular reason `Repository` is capitalized here?

Author: No reason, suggestion merged.

}
```

> The `chainConfig` parameter within the `Config` structure is crucial when setting up an AnyTrust Orbit chain. It contains configurations that are distinct from those of a Rollup chain. The most significant parameter that sets apart an AnyTrust chain is the `DataAvailabilityCommittee`. This parameter plays a pivotal role in defining the nature of your chain.

Reviewer 2: ```suggestion
The `chainConfig` parameter within the `Config` structure lets you specify the following parameters:
```

In this example, `txReceipt` refers to the transaction receipt you received after deploying the AnyTrust chain. By inputting this receipt into the `createRollupPrepareTransactionReceipt` function, you can access comprehensive data about your deployment, including details about the core contracts and configuration settings.

### 4. Setting valid key set on parent chain:

> The final step is to set up a valid key set for your Data Availability Committee (DAC) on the parent chain. You can find more in our documentation about [ generating keys and the key set for your committee ]( /launch-orbit-chain/concepts/anytrust-orbit-chain-keyset-generation ). 

Author: Noted, thanks.

});
```

In this example, `upgradeExecutor_address` and `sequencerInbox_address` are placeholders for the Orbit chain's contract addresses. `keyset` is the key set you generated for your committee, and `deployer.address` refers to the owner's account address.

> Once you've created the transaction request using the above API, the next step is to sign and send the transaction. This transaction writes the key set to the parent chain, enabling it to recognize and verify the valid key set for your AnyTrust Orbit chain. 

Reviewer: ```suggestion
Once you've created the transaction request using the above API, the next step is to sign and send the transaction. This transaction writes the keyset to the parent chain, enabling it to recognize and verify the valid key set for your AnyTrust Orbit chain. 
```
